.gitlab/package_build/deb.yml

```diff
@@ -191,7 +191,7 @@ iot_agent_deb-armhf:
     - !reference [.on_packaging_change]
     - !reference [.on_go-version_change]
   stage: package_build
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_armhf$DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX:$DATADOG_AGENT_ARMBUILDIMAGES
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_armhf$CI_IMAGE_DEB_ARM64_SUFFIX:$CI_IMAGE_DEB_ARM64_VERSION
```

Comment:
This should use a dedicated tag for armhf instead of arm64 as both are different images